src/FloatingWindows/FLWIntegration.cpp

```diff
@@ -25,6 +25,8 @@
 
 namespace flwnd {
 
+char window_cstring[250];
```

Comment:
Fixed-length C string in C++ code is inviting problems more often than not. Eventually someone will try to write a long message into it and corrupt memory, which is a kind of error very hard to debug.

Using std::string and [std::stringstream](http://www.cplusplus.com/reference/sstream/stringstream/) to build them is a much safer path.